libs/state/effects/src/lib/effects.service.ts

```diff
@@ -91,6 +83,16 @@ export class RxEffects implements OnDestroy$ {
   onDestroy$: Observable<boolean> = this._hooks$.pipe(toHook('destroy'));
   private readonly destroyers: Record<number, Subject<void>> = {};
 
+  constructor(
+    @Optional()
+    private readonly errorHandler: ErrorHandler | null,
```

Comment:
What about using `inject` here as well?

Reply:
Done!

I somehow remembered that somewhere `RxEffects` is instantiated in a place using `new` - but i was wrong :)